### PR TITLE
Externalize stylesheet

### PR DESCRIPTION
**Pull request by Lauri-Nomme**
Move all styles to stylesheet.
Add option --stylesheet to output stylesheet.

(cherry picked from commit 109d361054f60c6e192202aa74ec413559a54c9d)

### CODE REVIEW

**Comment by ptrovatelli** [comment]
hello @Lauri-Nomme ; thank you for all your contributions. i'll have a look at it within a couple of weeks, starting with the ones that don't have merge conflicts; after merging those, i'll come back to you for conflicts resolution :)

File: index.js [hunk 1]
```diff
@@ -87,6 +91,12 @@ function logError(context, error){
 }
 
 (async () => {
+    let stylesheet = await fs.readFile(__dirname + "/style.css", "binary");
+    if (argv.stylesheet) {
```
Comment (thecampagnards) [inline index.js:95]: I think you forgot to push your last changes to handle the arg

**Review by thecampagnards — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (Lauri-Nomme) [reply index.js:95]: hmm but isnt the arg handled in this very if?

Comment (Lauri-Nomme) [reply index.js:95]: if you could provide some guidance as to what i should change, that would be awesome.

**Comment by thecampagnards** [comment]
I will test it tomorrow and merge it if it's okey

**Comment by thecampagnards** [comment]
Tested and merged

**Pull request closed without merging**